vault/dendron.ref.commands.goto.md

```diff
@@ -0,0 +1,15 @@
+---
+id: vikda112zw2sss8lbit4qey
+title: Go to
+desc: ''
+updated: 1651331545738
+created: 1651331147246
+shortcuts: none
+---
+
+- shortcuts: {{fm.custom.shortcuts}}
+
+Go to the current link under cursor. This command will exhibit different behavior depending on the type of the link:
+- for wikilinks to [[proxy notes|dendron://dendron.dendron-site/dendron.topic.special-notes#proxy-notes]], this command will open the link designated by the `uri` property of the frontmatter
```

Comment:
nit: change the ordering of these bullet points to put this one at the bottom.  I'd expect proxy notes usecase to be by far the least of these 3.